views/dashboard.hbs

```diff
@@ -0,0 +1,107 @@
+            
+        <!-- font awesome -->
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" />
+
+        <!-- Google font  -->
+		<link href="https://fonts.googleapis.com/css?family=Jua|Slabo+27px" rel="stylesheet">
+		
+        <!-- jQuery library -->
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+
+        <!-- Latest compiled JavaScript -->
+        <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>	
+		
+</head>
+<body class="container-fluid">
+    <div class="contentBox" style="">
+        <div id="toolbar" >
+            <!-- Row 1-->
+            <!-- For some reason the BGcolor was not working from CSS side i will need help here -->
+            <div class="row toolbar-parent" style="background-color: white">
+                <!-- udacity Logo i cannot make it responsive so TODO fix this-->
+                <div class="col-md-6 UdaLogo"><img src="https://i0.wp.com/makemeflow.org/advice/wp-content/uploads/2016/03/udacity.png?fit=600%2C300&ssl=1" alt=""align="left" height="50"></div>
+                <!-- TODO Later convert these into buttons-->
+                <div class="col-md-1 toolbar-elements">Dashboard</div>
+                <div class="col-md-1 toolbar-elements">News Feed</div>
+                <div class="col-md-1 toolbar-elements">Network</div>
+                <div class="col-md-1 toolbar-elements">Resouces</div>
+                <div class="col-md-1 toolbar-elements">Profile</div>
+                <!-- dropdown button dummy -->
+                <!-- add and arrow svg or png to show the dropdown properly-->
+                <div class="col-md-1 toolbar-elements dropdown show">
+                    <a class="btn btn-outline-light dropdown-toggle" href="#" role="button" id="dropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"> ></a>
+                    <div class="dropdown-menu" aria-labelledby="dropdownMenuLink">
+                        <a class="dropdown-item" href="#">Action</a>
+                        <a class="dropdown-item" href="#">Another action</a>
+                        <a class="dropdown-item" href="#">Something else here</a>
+                    </div>
+                </div>
+            </div>
+            <br> 
+            <!-- TaskManager-->
+            <div id="taskManager">
+                <!-- Row 2 -->
+            <div class="row">
+                <!-- Task Manager container TODO give ids to these buttons -->
+                <div class="col-md-2 taskManagerContainer">
+                    <button type="button" class="buttonPadding btn btn-outline-light btn-lg btn-block">   Blog Post  </button>
+                    <button type="button" class="buttonPadding btn btn-outline-light btn-lg btn-block">   Add Projects</button>
```

Comment:
any specific reason for using buttonPadding class. As far as I understand we should use general css to avoid discrepencies

Reply:
To give the button a slight shadow, and a proper white background

so this is the class 
'css
.buttonPadding{ 
	background: white;
	display: flex;
	margin-left: 1%;
	margin-top: 5%;
	padding: 50px; 
	box-shadow: 0px 1px 4px 0px rgba(0, 0, 0, 0.35);
}
'

Reply:
class says padding to button but it contains a lot of other css properties. I will take a look and will suggest some changes, if that's fine to you

Reply:
@SMahil 
It is slightly suggested that do not do anything to this as i will probably be re basing my local to the master and redoing it with Material.css seems like the right thing to do